ui/core-react/src/core-react/searchbox/SearchBox.tsx

```diff
@@ -182,3 +175,28 @@ export class SearchBox extends React.Component<SearchBoxProps, SearchBoxState> {
     if (this._inputElement) this._inputElement.focus();
   }
 }
+
+const SearchBoxInput = React.forwardRef<
+  HTMLInputElement,
+  React.ComponentProps<typeof Input>
+>(function SearchBoxInput({ placeholder, ...props }, ref) {
+  const { translate } = useTranslation();
+  return (
+    <Input
+      ref={ref}
+      placeholder={placeholder ? placeholder : translate("general.search")}
```

Comment:
```suggestion
      placeholder={placeholder ?? translate("general.search")}
```

Reply:
Same as Dialog.tsx https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABFApgZygCgwJwFyK4xgDmiAPouACYrDErUCUiA3gFDuKIQIaIAPRAF5CUHIgD8YiQQBEtYAEMQAGyhyA3Fx58oiAJ4iZU6Qror1WnbzBo4qlADpVcEpg7duAgDSGdAL5M2gGcqBiYckoARhBywezhWDR0DMzaSZHxmogA9LmI1DDAwCg4KGBQQA

Reply:
In that case we can use `||`. I just want this to be more clean

Reply:
Replaced with logical OR.